### ISSUE TITLE
Keyboard access Failing Examples

### ISSUE THREAD

**Issue by smh243** [Section508Coordinators/DHSA-TT-200-A#111]
If you haven't already, please review the [Guidelines for Contributing](contributing.md) in the main folder of this repository before submitting any issues.

## Issue Type (put an X between the brackets to indicate issue type)
- [ x] System Bug (the course doesn't function properly)
- [ ] Typo/grammar
- [ ] Incorrect content
- [ ] 508 issue (something is not accessible according to our own test process)
- [ ] Other, specify...[replace this text with your issue type description]

## Description
[Description of the issue]

## Location
https://training.section508testing.net/mod/book/view.php?id=1229&chapterid=2676
Lesson 1: Keyboard Access
Failing examples
Example 2 

## Actual Behavior / Observation
Opened in Google Chrome/Win 10. When mousing over the links there was no change in the mouse pointer to indicate there are active components on the page.

Also in the evaluation results step 1, it says four interactive elements cannot be accessed. I could only find 3 the 3 buttons shown between steps 3 and 4.

**Comment by drewnielson** [comment]
Thanks! Default mouse pointer is the same pointer for buttons. However, fixed to display finger-pointer on mouseover for the links. Also fixed text in Evaluation Results step 1 to refer to only 3 interactive elements that cannot be accessed by keyboard.
